specifications/dpe-irot-profile/spec.ocp

```diff
@@ -1306,7 +1294,7 @@ The format “ocp.csr.irot-eca.mldsa87” is defined as follows:
 +-------------------------------+------------------------------------------------------------------+
 | =========================================== Export ============================================= |
 +-------------------------------+------------------------------------------------------------------+
-| export-cdi-format             | ocp.export-cdi.raw-512                                           |
+| export-cdi-format             | ocp.export-cdi.raw-384                                           |
```

Comment:
Same as above. Maybe we need separate sized parameters for a measurement hash v.s. a CDI?